source/fundamentals/crud/read-operations/cursor.txt

```diff
@@ -13,13 +13,15 @@ Access Data From a Cursor
 Overview
 --------
 
-In this guide, you can learn how to access data using a cursor with the
+In this guide, you can learn how to access data using a **cursor** with the
 MongoDB Java driver. 
 
-Read operations that return multiple documents do not immediately return
-all values matching the query. Because a query can potentially match
-a large number of documents, we need to be able to access or store the
-matched documents. 
+A cursor is a tool for iterating over database results while only holding a
+subset of them in memory at a given time.
+
+Read operations that match a large number of documents do not return
```

Comment:
Issue: The transition here seems a bit rough.  The first sentence here seems to be reiterating the paragraph right before. I think this paragraph should be appended to the previous paragraph to explain how the cursor is used.